app/src/main/java/com/example/harvestfresh/CartAdapter.java

```diff
@@ -66,7 +60,7 @@ public ViewHolder(@NonNull View itemView) {
             super(itemView);
             tvProduct = itemView.findViewById(R.id.tvProduct);
             tvPrice = itemView.findViewById(R.id.tvPrice);
-            ibRemove = itemView.findViewById(R.id.ibRemove);
+            ibRemove = itemView.findViewById(R.id.ibAdd);
```

Comment:
This looks sketchy, whats going on? Do we need to update this variable name to ibAdd, or do we need to add a second variable for ibAdd?

Reply:
the Button id was incorrectly named ibAdd instead of remove, thanks for the catch!